sylviawanjiku/StoreManager
# #161862011 Consume Get all products endpoints

#### What does this PR do?
Consumes the GET Request for all products
#### Description of Task to be completed?
A user can get to view  alist of all products from the interface
#### How should this be manually tested?
On the store manager UI the user should click the view products buttton to load alist of products
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#161862011
https://www.pivotaltracker.com/story/show/161862011
#### Screenshots (if appropriate)
#### Questions:
